pynicotine/gtkgui/ui/uploads.ui

```diff
@@ -176,16 +148,16 @@
                     <property name="visible">True</property>
                     <property name="spacing">6</property>
                     <child>
-                      <object class="GtkLabel">
+                      <object class="GtkImage">
                         <property name="visible">True</property>
-                        <property name="label" translatable="yes">Clear Uploa_ds…</property>
```

Comment:
I'm fine with giving the button removal a go, but I'd prefer if we kept the arrow on menu buttons for consistency.

Reply:
It looks a bit too funky with both the Eject icon and the Pan-Up icon, so I have reverted the button to how it was before, with the Pan-Up icon on the right side as is standard for a menu button.